manifests/init.pp

```diff
@@ -273,7 +273,7 @@
   include pulpcore::plugin::certguard # Required to be present by Katello when syncing a content proxy
 
   class { 'foreman_proxy::plugin::pulp':
-    enabled              => !$pulpcore_mirror,
+    enabled              => !$pulpcore_mirror and $facts['os']['release']['major'] == '7',
```

Comment:
I think this would create an issue for non-RH operating systems. What should the behavior be here for those users?

Reply:
This module doesn't support non-RH OSes so it's not really a factor.

Reply:
This isn't supported anywhere but EL operating systems so we are good.